src/components/CurrencyInputPanel/SwapCurrencyInputPanel.tsx

```diff
@@ -67,8 +67,8 @@ const CurrencySelect = styled(ButtonGray)<{
   opacity: ${({ disabled }) => (!disabled ? 1 : 0.4)};
   color: ${({ selected, theme }) => (selected ? theme.neutral1 : theme.white)};
   cursor: pointer;
-  height: unset;
-  border-radius: 16px;
+  height: 36px;
+  border-radius: 32px;
```

Comment:
why is the radius so large? does it have any effect beyond 18px?

Reply:
no it could even be 50% i think to be fancy, i get used to the RN world of no fancy css stuff sometimes.